client_config/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
     image: redis/redis-stack-server:latest
 
   chroma:
-    image: ghcr.io/chroma-core/chroma:0.4.11
+    image: ghcr.io/chroma-core/chroma:0.4.12
```

Comment:
Any reason for `0.4.12` vs `0.4.13` (latest)?

https://github.com/chroma-core/chroma/releases/tag/0.4.13
https://github.com/chroma-core/chroma/pkgs/container/chroma/131289546?tag=0.4.13

Reply:
Only upgrading to required version to solve this error, but I can change this.